README.md

```diff
@@ -150,57 +152,68 @@ You can check out the recorded demo to see how the navigator works.
 ## Publishing to the Chrome Web Store
```

Comment:
Looking at the page altogether, it's getting a bit long.  The home page serves in part as a marketing page for your work, and for that purpose, shorter is better.   I'd suggest moving the publishing content to a separate page under `docs` and linking from the README.